<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>MainWindow</class>
 <widget class="QMainWindow" name="MainWindow">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>575</width>
    <height>401</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>营养计算器</string>
  </property>
  <widget class="QWidget" name="centralWidget">
   <widget class="QTabWidget" name="tabWidget">
    <property name="geometry">
     <rect>
      <x>0</x>
      <y>0</y>
      <width>571</width>
      <height>331</height>
     </rect>
    </property>
    <property name="currentIndex">
     <number>1</number>
    </property>
    <widget class="QWidget" name="tab">
     <attribute name="title">
      <string>程序化咨询</string>
     </attribute>
     <widget class="QLabel" name="label">
      <property name="geometry">
       <rect>
        <x>50</x>
        <y>10</y>
        <width>201</width>
        <height>31</height>
       </rect>
      </property>
      <property name="font">
       <font>
        <pointsize>13</pointsize>
       </font>
      </property>
      <property name="text">
       <string>您是否曾被诊断为糖尿病:</string>
      </property>
     </widget>
     <widget class="QLabel" name="label_2">
      <property name="geometry">
       <rect>
        <x>50</x>
        <y>90</y>
        <width>201</width>
        <height>31</height>
       </rect>
      </property>
      <property name="font">
       <font>
        <pointsize>13</pointsize>
       </font>
      </property>
      <property name="text">
       <string>您的年龄是:</string>
      </property>
     </widget>
     <widget class="QGroupBox" name="groupBox">
      <property name="enabled">
       <bool>true</bool>
      </property>
      <property name="geometry">
       <rect>
        <x>80</x>
        <y>40</y>
        <width>401</width>
        <height>51</height>
       </rect>
      </property>
      <property name="title">
       <string/>
      </property>
      <widget class="QRadioButton" name="radioButton">
       <property name="enabled">
        <bool>true</bool>
       </property>
       <property name="geometry">
        <rect>
         <x>30</x>
         <y>10</y>
         <width>108</width>
         <height>23</height>
        </rect>
       </property>
       <property name="text">
        <string>是</string>
       </property>
       <property name="checked">
        <bool>true</bool>
       </property>
       <attribute name="buttonGroup">
        <string notr="true">buttonGroup</string>
       </attribute>
      </widget>
      <widget class="QRadioButton" name="radioButton_2">
       <property name="enabled">
        <bool>true</bool>
       </property>
       <property name="geometry">
        <rect>
         <x>270</x>
         <y>10</y>
         <width>108</width>
         <height>23</height>
        </rect>
       </property>
       <property name="text">
        <string>否</string>
       </property>
       <attribute name="buttonGroup">
        <string notr="true">buttonGroup</string>
       </attribute>
      </widget>
     </widget>
     <widget class="QGroupBox" name="groupBox_2">
      <property name="geometry">
       <rect>
        <x>90</x>
        <y>120</y>
        <width>381</width>
        <height>31</height>
       </rect>
      </property>
      <property name="title">
       <string/>
      </property>
      <widget class="QRadioButton" name="radioButton_3">
       <property name="geometry">
        <rect>
         <x>20</x>
         <y>10</y>
         <width>108</width>
         <height>23</height>
        </rect>
       </property>
       <property name="text">
        <string>&lt;=35</string>
       </property>
       <property name="checked">
        <bool>true</bool>
       </property>
       <attribute name="buttonGroup">
        <string notr="true">buttonGroup_2</string>
       </attribute>
      </widget>
      <widget class="QRadioButton" name="radioButton_4">
       <property name="geometry">
        <rect>
         <x>260</x>
         <y>10</y>
         <width>108</width>
         <height>23</height>
        </rect>
       </property>
       <property name="text">
        <string>&gt;36</string>
       </property>
       <attribute name="buttonGroup">
        <string notr="true">buttonGroup_2</string>
       </attribute>
      </widget>
     </widget>
     <widget class="QLineEdit" name="userhight">
      <property name="geometry">
       <rect>
        <x>140</x>
        <y>260</y>
        <width>113</width>
        <height>28</height>
       </rect>
      </property>
     </widget>
     <widget class="QLineEdit" name="userweight">
      <property name="geometry">
       <rect>
        <x>360</x>
        <y>260</y>
        <width>113</width>
        <height>28</height>
       </rect>
      </property>
     </widget>
     <widget class="QLabel" name="label_4">
      <property name="geometry">
       <rect>
        <x>60</x>
        <y>260</y>
        <width>61</width>
        <height>21</height>
       </rect>
      </property>
      <property name="text">
       <string>身高(cm):</string>
      </property>
     </widget>
     <widget class="QLabel" name="label_5">
      <property name="geometry">
       <rect>
        <x>290</x>
        <y>260</y>
        <width>61</width>
        <height>21</height>
       </rect>
      </property>
      <property name="text">
       <string>体重(kg):</string>
      </property>
     </widget>
     <widget class="QLabel" name="label_6">
      <property name="geometry">
       <rect>
        <x>50</x>
        <y>160</y>
        <width>201</width>
        <height>31</height>
       </rect>
      </property>
      <property name="font">
       <font>
        <pointsize>13</pointsize>
       </font>
      </property>
      <property name="text">
       <string>是否肾功能受损:</string>
      </property>
     </widget>
     <widget class="QGroupBox" name="groupBox_3">
      <property name="enabled">
       <bool>true</bool>
      </property>
      <property name="geometry">
       <rect>
        <x>80</x>
        <y>190</y>
        <width>401</width>
        <height>51</height>
       </rect>
      </property>
      <property name="title">
       <string/>
      </property>
      <widget class="QRadioButton" name="radioButton_5">
       <property name="enabled">
        <bool>true</bool>
       </property>
       <property name="geometry">
        <rect>
         <x>30</x>
         <y>10</y>
         <width>108</width>
         <height>23</height>
        </rect>
       </property>
       <property name="text">
        <string>是</string>
       </property>
       <property name="checked">
        <bool>true</bool>
       </property>
      </widget>
      <widget class="QRadioButton" name="radioButton_6">
       <property name="enabled">
        <bool>true</bool>
       </property>
       <property name="geometry">
        <rect>
         <x>270</x>
         <y>10</y>
         <width>108</width>
         <height>23</height>
        </rect>
       </property>
       <property name="text">
        <string>否</string>
       </property>
      </widget>
     </widget>
    </widget>
    <widget class="QWidget" name="tab_2">
     <attribute name="title">
      <string>食物列举</string>
     </attribute>
     <widget class="QTableWidget" name="tableWidget_2">
      <property name="geometry">
       <rect>
        <x>0</x>
        <y>0</y>
        <width>451</width>
        <height>281</height>
       </rect>
      </property>
      <row>
       <property name="text">
        <string>1</string>
       </property>
      </row>
      <row>
       <property name="text">
        <string>2</string>
       </property>
      </row>
      <row>
       <property name="text">
        <string>3</string>
       </property>
      </row>
      <row>
       <property name="text">
        <string>4</string>
       </property>
      </row>
      <row>
       <property name="text">
        <string>5</string>
       </property>
      </row>
      <row>
       <property name="text">
        <string>7</string>
       </property>
      </row>
      <column>
       <property name="text">
        <string>食物名称</string>
       </property>
      </column>
      <column>
       <property name="text">
        <string>A</string>
       </property>
      </column>
      <column>
       <property name="text">
        <string>B</string>
       </property>
      </column>
      <column>
       <property name="text">
        <string>脂肪</string>
       </property>
      </column>
     </widget>
    </widget>
    <widget class="QWidget" name="tab_3">
     <attribute name="title">
      <string>食物查询</string>
     </attribute>
     <widget class="QTextBrowser" name="textBrowser">
      <property name="geometry">
       <rect>
        <x>90</x>
        <y>100</y>
        <width>381</width>
        <height>141</height>
       </rect>
      </property>
     </widget>
     <widget class="QLineEdit" name="lineEdit">
      <property name="geometry">
       <rect>
        <x>90</x>
        <y>60</y>
        <width>261</width>
        <height>28</height>
       </rect>
      </property>
     </widget>
     <widget class="QPushButton" name="pushButton">
      <property name="geometry">
       <rect>
        <x>380</x>
        <y>60</y>
        <width>90</width>
        <height>28</height>
       </rect>
      </property>
      <property name="text">
       <string>查询</string>
      </property>
     </widget>
     <widget class="QLabel" name="label_3">
      <property name="geometry">
       <rect>
        <x>20</x>
        <y>110</y>
        <width>58</width>
        <height>18</height>
       </rect>
      </property>
      <property name="text">
       <string>结果</string>
      </property>
     </widget>
    </widget>
   </widget>
  </widget>
  <widget class="QMenuBar" name="menuBar">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>0</y>
     <width>575</width>
     <height>23</height>
    </rect>
   </property>
   <widget class="QMenu" name="menu">
    <property name="title">
     <string>开始</string>
    </property>
    <addaction name="action_2"/>
   </widget>
   <widget class="QMenu" name="menu_2">
    <property name="title">
     <string>查询</string>
    </property>
    <addaction name="action"/>
   </widget>
   <widget class="QMenu" name="menu_3">
    <property name="title">
     <string>退出</string>
    </property>
   </widget>
   <addaction name="menu"/>
   <addaction name="menu_2"/>
   <addaction name="menu_3"/>
  </widget>
  <widget class="QToolBar" name="mainToolBar">
   <attribute name="toolBarArea">
    <enum>TopToolBarArea</enum>
   </attribute>
   <attribute name="toolBarBreak">
    <bool>false</bool>
   </attribute>
  </widget>
  <widget class="QStatusBar" name="statusBar"/>
  <action name="action">
   <property name="text">
    <string>新建查询</string>
   </property>
  </action>
  <action name="action_2">
   <property name="text">
    <string>打开</string>
   </property>
  </action>
 </widget>
 <layoutdefault spacing="6" margin="11"/>
 <resources/>
 <connections/>
 <buttongroups>
  <buttongroup name="buttonGroup"/>
  <buttongroup name="buttonGroup_2"/>
 </buttongroups>
</ui>
